lib/ansible/modules/storage/ibm/ibm_sa_host.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (C) 2018 IBM CORPORATION
+# Author(s): Tzur Eliyahu <tzure@il.ibm.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.1'}
+
+DOCUMENTATION = '''
+---
+module: ibm_sa_host
+short_description: Adds hosts to or removes them from IBM Spectrum Accelerate storage systems.
+version_added: "2.7"
+
+description:
+    - "This module adds hosts to or removes them from IBM Spectrum Accelerate storage systems."
+
+options:
+    host:
+        description:
+            - Host name.
+        required: true
+    state:
+        description:
+            - Host state.
+        required: true
+        default: "present"
+        choices: [ "present", "absent" ]
+    cluster:
+        description:
+            - The name of the cluster to include the host.
+        required: false
+    domain:
+        description:
+            - The domains the cluster will be attached to.
+                To include more than one domain,
+                separate domain names with commas.
+                To include all existing domains, use an asterisk ("*").
+        required: false
+    iscsi_chap_name:
+        description:
+            - The host's CHAP name identifier
+        required: false
+    iscsi_chap_secret:
+        description:
+            - The password of the initiator used to
+                authenticate to the system when CHAP is enable
+        required: false
+
+extends_documentation_fragment:
+    - ibm_storage
+
+author:
+    - Tzur Eliyahu (tzure@il.ibm.com)
+'''
+
+EXAMPLES = '''
+- name: Define new host.
+  ibm_sa_host:
+    host: host_name
+    state: present
+    username: admin
+    password: secret
+    endpoints: hostdev-system
+
+- name: Delete host.
+  ibm_sa_host:
+    host: host_name
+    state: absent
+    username: admin
+    password: secret
+    endpoints: hostdev-system
+'''
+RETURN = '''
```

Comment:
What is the return scheme?

Reply:
This should only return an indication of if the define process succeeded or not. No need to add any more value or Ansible facts.